### PR TITLE
AB#6612 Implement vendor and speaker page new designs

### PR DESCRIPTION
**Pull request by jflheureux**

### CODE REVIEW

File: Website/src/rendering/src/temp/GraphQLIntrospectionResult.json [hunk 1]
```diff
@@ -1240,6 +1240,11 @@
             "name": "RichTextField",
             "ofType": null
           },
+          {
```
Comment (adoprog) [inline Website/src/rendering/src/temp/GraphQLIntrospectionResult.json:1243]: Is this file required? I am confused by the folder name "temp".

**Review by adoprog — APPROVED** [review]
Nice work! Checked local and AKS.
Only one note: the new InformationPageHero is not editable in Horizon/EE (only name is editable). I assume it is not important as the content is from CH.

Storybook Pages look great!

File: Website/src/rendering/src/components/InformationPageHero.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                Meet the <span className="information-type">{props.qualificative}</span>{' '}
+                {props.type}:
+              </p>
+              <h1 className="name">
+                <Text field={props.fields.Name} />
+              </h1>
+            </div>
+            {informations}
+            <div className="external-website-icons">
+              {!props.fields.FacebookProfileLink ? (
```
Comment (jflheureux) [inline Website/src/rendering/src/components/InformationPageHero.tsx:51]: Switch to Alexander's new component

File: Website/src/rendering/src/components/SpeakerInformationPageHero.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  };
+  const qualificative = props.fields.Featured.value ? 'featured' : '';
+
+  const informations =
+    props.fields.JobTitle?.value || props.fields.Company?.value || props.fields.Location?.value ? (
+      <>
+        {props.fields.JobTitle?.value ? (
+          <div>
+            <span className="label">Job Title:</span>{' '}
+            <Text field={props.fields.JobTitle} tag="span" />
```
Comment (jflheureux) [inline Website/src/rendering/src/components/SpeakerInformationPageHero.tsx:38]: Ensure the informations are editable in Experience Editor

File: Website/src/rendering/src/components/InformationPageHero.tsx [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+        <div className="gradient-container"></div>
+        <div className="content-container">
+          <div className="container-content-text">
+            <div>
+              <p className="title">
+                Meet the <span className="information-type">{props.qualificative}</span>{' '}
+                {props.type}:
+              </p>
+              <h1 className="name">
+                <Text field={props.fields.Name} />
```
Comment (jflheureux) [inline Website/src/rendering/src/components/InformationPageHero.tsx:46]: Ensure the name is editable in Experience Editor

File: Website/src/rendering/src/components/SpeakerInformation.tsx [hunk 5]
```diff
@@ -1,74 +1,43 @@
...
-          </div>
           <div className="description-col">
-            <div>
-              <Text tag="div" className="eyebrow" field={props.fields.Role} />
-              {featuredEyebrow}
-            </div>
-            <Text tag="h1" className="name" field={props.fields.Name} />
-            <RichText field={props.fields.Description} />
+            <div className="column-title">Biography:</div>
+            <RichText field={props.fields?.data?.contextItem?.description} />
```
Comment (jflheureux) [inline Website/src/rendering/src/components/SpeakerInformation.tsx:37]: Ensure the description is editable in Experience Editor

File: Website/src/rendering/src/temp/GraphQLIntrospectionResult.json (hunk 1 above)
Comment (jflheureux) [reply Website/src/rendering/src/temp/GraphQLIntrospectionResult.json:1243]: Yes, this file is being used by JSS GraphQL client. You can read about it here: https://jss.sitecore.com/docs/nextjs/graphql/introspection

File: Website/src/rendering/src/components/SpeakerInformation.tsx (hunk 5 above)
Comment (jflheureux) [reply Website/src/rendering/src/components/SpeakerInformation.tsx:37]: Unable to make description editable as it comes from GraphQL
